TRANSLATION.md

```diff
@@ -37,10 +37,11 @@ Hierdoor zal bovenstaande link niet meer werken.
 Laat tekst in codeblokken onvertaald, behalve voor commentaar. Je kan eventueel tekst in strings vertalen, maar let op dat je geen strings vertaalt die naar code verwijzen!
 
 Voorbeeld:
+
 ```js
 // Example
 try {
-  const product = await updateProjectMutation({ name: 'Cool Shoes' })
+  const product = await updateProjectMutation({ name: "Cool Shoes" })
```

Comment:
Is het in Blitz conventie om dubbele quotes te gebruiken voor strings in objecten?

Reply:
Ik denk het wel, want in hun prettier config hebben ze `singleQuote: false` staan.